core/lib/globals_shared.h

```diff
@@ -2017,4 +2017,30 @@ typedef struct _priv_mcontext_t {
 #include "mcxtx.h"
 } priv_mcontext_t;
 
+/* DR_API EXPORT BEGIN */
+/**
+ * Encodings of an application's argument.
+ */
+typedef enum {
+    APP_ARG_ASCII,
+    APP_ARG_UTF_8,
+    APP_ARG_UTF_16,
+} dr_app_arg_encoding_t;
+
+typedef struct _dr_app_arg_t {
```

Comment:
Missing doxygen comment (so it won't show up in the docs)

Reply:
Done.